dataflow/tests/busy-expression/Expected.txt

```diff
@@ -0,0 +1,295 @@
+2 -> 3 EACH_TO_EACH
```

Comment:
This file is in yet another variant of `busy-expression`. I think it would be nicer if this matches the same package name.

Reply:
okay